deploy/cr-apm.yaml

```diff
@@ -51,3 +51,14 @@ spec:
   # Sets a NetworkZone for the OneAgent (optional)
   # Note: This feature requires OneAgent version 1.195 or higher
   #networkZone: name-of-my-network-zone
+
+  # Optional: resource settings for initContainer. Consumption of the OneAgent heavily depends
```

Comment:
Not really, this the consumption is pretty much static. This is only for the unzipping.

Well, I guess if your OneAgent package is bigger, the unzipping can need more CPU.